package.json

```diff
@@ -45,51 +45,53 @@
     "access": "public"
   },
   "dependencies": {
-    "@sasjs/adapter": "^2.2.18",
-    "@sasjs/core": "^2.25.0",
-    "@sasjs/lint": "^1.6.1",
-    "@sasjs/utils": "^2.12.1",
-    "btoa": "^1.2.1",
-    "chalk": "^4.1.0",
-    "csv-stringify": "^5.6.1",
-    "dotenv": "^8.2.0",
-    "esm": "^3.2.25",
-    "find": "^0.3.0",
+    "@sasjs/adapter": "2.2.18",
```

Comment:
The package lock file needs to be regenerated as well.

Reply:
`package.json` and `package-lock.json` are synced